src/components/Dropdown/__tests__/__snapshots__/Dropdown.test.tsx.snap

```diff
@@ -1279,6 +1351,9 @@ exports[`Dropdown displays all options when focused 1`] = `
         </span>
       </div>
     </div>
+    <div
```

Comment:
We shouldn't render this hidden options container if the flag for being space aware is false - and add a snapshot to capture it